### ISSUE TITLE
Calling `Waku.stop` does not clear all timers

### ISSUE THREAD

**Issue by felicio** [status-im/js-waku#766]
This is a **bug report**

## Problem

Calling `Waku.stop` does not clear all timers, thus preventing a process from exiting, or otherwise finishing, immediately and eventually allowing use of closed connections and `throw`ing.

Source:
```js
// example.js
/**
 * Source code example
 */

'use strict'

const { Waku } = require('js-waku')

;(async () => {
  const waku = await Waku.create({
    bootstrap: {
      default: true,
    },
  })
  await waku.waitForRemotePeer()

  await waku.stop()
})()
```

Run:
```shell
node example.js
```

---
`js-waku`: `0.24.0`
`node`: `16.13.1`

## Proposed Solutions

On `Waku.stop`, call `clearInterval` for all `pingKeepAliveTimers` and `relayKeepAliveTimers`.

## Notes

Relates to https://github.com/libp2p/js-libp2p/issues/939, as the `peer:disconnect` event is never emitted and a single peer's timer cannot be cleared. However, in case of multiple peers even that event would not help, because in that case it is supposed to be fired only for the last closed connection.

**Comment by D4nte** [comment]
Thanks! How did you notice?

**Comment by felicio** [comment]
> Thanks! How did you notice?

@D4nte

Tldr: debugger within vscode would not exit.

I was doing a small research on what events a client should implement in terms of connectivity (eg. connected, disconnected, errors), how it should indicate its states to users and how it should clean up after itself.

I'd expect Waku to handle and propagate some of those events, but that's something I'd like to discuss some other time. But about the cleanup, vscode's debugger nor node would exit, so I used `wtfnode.dump()` to see what was blocking.
